tests/apm_tracing_e2e/test_otel.py

```diff
@@ -0,0 +1,76 @@
+from tests.apm_tracing_e2e.test_single_span import _get_spans_submitted, _assert_msg
+from utils import context, weblog, scenarios, missing_feature, irrelevant
+
+
+@missing_feature(
+    context.weblog_variant not in "net-http",
```

Comment:
Should the `"net-http"` be a list/tuple instead of a string?

Reply:
Works both ways 🙌